openapi.yaml

```diff
@@ -943,6 +942,11 @@ components:
         - supervisor
         - effectiveDate
       properties:
+        employmentType:
```

Comment:
Do we still want this field? I guess we can also use `studentEmployeeInd` to replace this?

Reply:
Oh yeah I guess studentEmployeeInd can replace it entirely.